### ISSUE TITLE
Raw Dataset

### ISSUE THREAD

**Issue by SafaaDaf** [MarlonCajamarca/Keras-LSTM-Trajectory-Prediction#2]
Thank you for your great work. 

Could you please tell me the structure of the raw dataset 'csv' before converting in it to hdf5 file ? So I can use afterthat my own dataset.

Thank you.

**Comment by MarlonCajamarca** [comment]
Hi SafaaDaf

Firstly, sorry for the late reply! Actually, I am using Yolo-V3 detections grouped by a track  ID. Each track contains the set of detections (darknet detections) for the same object, forming its trajectory. Each raw input trajectory is then a sequence of detections that are then preprocessed to create the training and test datasets.

Keep an eye into the repository, I will update it with better scripts to preprocess and train LSTM recurrent cells (using Ranger optimizer and bidirectional wrapper to increase speed of convergence and lower loss and metric results!

**Comment by MarlonCajamarca** [comment]
Hi SafaaDaf

Firstly, sorry for the late reply! Actually, I am using Yolo-V3 detections grouped by a track  ID. Each track contains the set of detections (darknet detections) for the same object, forming its trajectory. Each raw input trajectory is then a sequence of detections that are then preprocessed to create the training and test datasets.

Keep an eye into the repository, I will update it with better scripts to preprocess and train LSTM recurrent cells (using Ranger optimizer and bidirectional wrapper to increase speed of convergence and lower loss and metric results!

**Comment by SafaaDaf** [comment]
Thank you for your response. Could you please provide me with the script that you used to track the object by ID and to group them in one file with the detections ? I would be greatful.

**Comment by SafaaDaf** [comment]
Hi MarlonCajamarca, 

Thank you for updating the repository it's interesting. I won't be demanding, but could you please upload the csv file containing the raw dataset or the code used to create this csv file ? 

Thank you in advance.

**Comment by wanghuohuo0716** [comment]
Hi MarlonCajamarca,
Thank you for your great work！

could you please upload the .csv or .txt dataset？

thanks!

**Comment by MarlonCajamarca** [comment]
Samples for the datasets used for training and evaluation have been linked into the README.md file.
